data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/contract.verification/CordaPackageSummary.avsc

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "record",
+  "name": "CordaPackageSummary",
+  "namespace": "net.corda.ledger.utxo.contract.verification",
+  "fields": [
+    {
+      "name": "name",
+      "type": "string",
+      "doc": "Name of the package."
+    },
+    {
+      "name": "version",
+      "type": "string",
+      "doc": "Version of the package."
+    },
```

Comment:
This isn't including signer summary hash which is technically required to disambiguate the name. I presume mostly we will lookup Cpk's for the sandbox by file hash and then confirm that this aligns with the name, version, signer metadata